docs/lua-api/inventory.lua

```diff
@@ -29,6 +29,14 @@ function giveOrDropItems(player, ...) end
 --
 function takeItems(player, ...) end
 
+--- Set an item in the player inventory.
+-- @param player player
+-- @param slot slot of the inventory
+-- @param Item to set (each as an @{rpgplus.types.itemstack} or a string of the form `id:data:amount`)
+-- @return nil
```

Comment:
Anything that doesn't return a value returns `nil`, so you can leave this out. :wink: